curriculum/js2/5.js

```diff
@@ -4,8 +4,16 @@
  * @returns {number}
  */
 
-const solution = (a) => {
-  return 0;
+const solution = (a, idx = 1, largest = a[0]) => {
+  if (idx >= a.length) {
+    return a.length === 0 ? 0 : largest;
```

Comment:
`return largest || -1` Usually I use -1 for invalid input for functions that should return a number.